CONTRIBUTING.md

```diff
@@ -43,6 +43,10 @@ Or run just one test file
 
     npm run test:node -- --file=test/realtime/auth.test.js
 
+Or run just one test
```

Comment:
I'm happy to add this but it's nothing to do with incremental backoff so should be in a separate PR

Reply:
Okay, I might create a new PR for this : )

Reply:
moved to #1364